src/actions/index.js

```diff
@@ -25,3 +25,5 @@ export const getMiningStatus = createAction('GET_MINING_STATUS', getMiningStatus
 export const getTransactionCount = createAction('GET_TRANSACTION_COUNT', getTransactionCountCall);
 
 export const getLatestBlock = createAction('GET_LATEST_BLOCK', getLatestBlockCall);
+
+export const getRpcServer = createAction('GET_TOP_BLOCKS', getRpcServerCall);
```

Comment:
I feel as though 'GET_TOP_BLOCKS' is not the most appropriate name for this action. Maybe go with 'GET_RPC_SERVER' instead.